### ISSUE TITLE
feature: shorter logs for 'run'

### ISSUE THREAD

**Issue by akurdyukov** [meltano/meltano#8403]
### Feature scope

CLI (options, error messages, logging, etc.)

### Description

Current version logs every record synced on 'info' log level. Which leads to megabytes of logs with data disclosure on 'run' phase. I propose:

1. Move logs for records to 'debug' level
2. Add 'info' messages like 'X messages loaded' for every definable X records for visible progress

Maybe this should be implemented in Singer SDK?

**Comment by tayloramurphy** [comment]
@akurdyukov can you share which tap you're using (and any version), which version of meltano, and the command you're running? By default it shouldn't list every record. I suspect the tap is logging that at the info level.

For an example of what the output should look like see https://docs.meltano.com/getting-started/part2#run-your-data-integration-el-pipeline which does show summary metrics as you'd like. cc @edgarrmondragon

**Comment by edgarrmondragon** [comment]
Right, a tap's stdout should only be emitted if debug logging is enabled: https://github.com/meltano/meltano/blob/b06f1ea1467d83daa703006b268e2ef06b61250c/src/meltano/core/block/extract_load.py#L613-L619

**Comment by akurdyukov** [comment]
I use standard tap-postgres. Lines I get are like this:

**Comment by edgarrmondragon** [comment]
@akurdyukov glad you figured it out. It was probably being caused by this bit in the logging config:

```yaml
root:
  level: DEBUG
  propagate: yes
  handlers: [console]
```
